spec/book_spec.rb

```diff
@@ -0,0 +1,31 @@
+require_relative '../book/book'
+
+describe Book do
+  context 'To create a new instance of the Book class we must ensure the following' do
+    it 'Return a Book object including publisher, cover_state and publish_date via constructor method' do
+      @publisher = 'Anek'
+      @cover_state = 'bad'
+      @publish_date = 2020
+      @book = Book.new(@publisher, @cover_state, @publish_date)
+      expect(@book.publisher).to eql @publisher
+      expect(@book.cover_state).to eql @cover_state
+      expect(@book.publish_date).to eql @publish_date
+    end
```

Comment:
I suggest you assign `@cover_state`, `@publisher`, and `@publish_date` to  `expected_value` and call the `expected_value` inside eq. 
```
Example:
(
expected_value = @publisher
expect (@book.publisher).to eql(expected_value)
)
```